README.md

```
@@ -157,3 +176,5 @@ packages a runnable test case to attach.
 | **Discussions**| https://github.com/siliconcompiler/siliconcompiler/discussions
 | **Sources**|  https://github.com/siliconcompiler/siliconcompiler
 | **Issues**|  https://github.com/siliconcompiler/siliconcompiler/issues
+| **Open PDKs**| https://github.com/siliconcompiler/lambdapdk — every open PDK and standard cell library SiliconCompiler ships
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)